fisher.fish

```diff
@@ -218,9 +218,21 @@ function _fisher_commit -a cmd
     if test "$cmd" = "rm"
         set out_pkgs $next_pkgs
     else
+        set -l current_pkg_name (command basename $argv)
+        set -l current_pkg_name_path $fisher_path/conf.d/$current_pkg_name.fish
+
         for pkg in $next_pkgs
             if contains -- (echo $pkg | command sed "s|@.*||") $actual_pkgs
                 set out_pkgs $out_pkgs $pkg
+
+                set -l pkg_name (command basename $pkg .fish)
+
+                if test "$pkg_name" = "$current_pkg_name"
```

Comment:
@jorgebucaran basically here I'm checking for emitting just once the event during installation.

Using your same `uninstall` approach on `_fisher_rm` function.

https://github.com/jorgebucaran/fisher/blob/c2b3ab01713256324720ee24ff2aa2da779a138d/fisher.fish#L399-L401

Reply:
Will this actually run only once? What happens when I run `fisher` again? See https://github.com/jorgebucaran/fisher/issues/527

Reply:
> Will this actually run only once?

Yes, when I say once I mean on every `fisher add foobar` only. 
Additionally, for example for new sessions created opening a new terminal tab, this event never gets emitted.

My concern was that I have no found a solution to make stuff on just every `fisher add foobar`.
Unless you have another more proper way to do it.

>  What happens when I run fisher again?

I have tried many times and I didn't see issues. I only saw that the `uninstall` event gets emitted at the beginning (related to #527), but then the normal `install` event gets emitted just at the end of the installation.

I have tested this PR with my package on local which has both events defined. Then performing `fisher add $PWD` many times as needed.

>  See #527

Is that already implemented or in progress?